File: src/main/java/com/cf/service/BbsReplyService.java
package com.cf.service;
import java.util.List;

import com.cf.pojo.TbBbsReply;
import com.cf.pojo.result.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface BbsReplyService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbBbsReply> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbBbsReply bbsReply,String topicUuid);
	
	
	/**
	 * 修改
	 */
	public void update(TbBbsReply bbsReply);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbBbsReply findOne(String id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbBbsReply bbsReply, int pageNum,int pageSize);
	
	/**
	 * 根据主题帖id查找下面的所有回复
	 * @param uuid
	 * @return
	 */
	public List<TbBbsReply> findByBbs(String uuid);
}

File: src/main/webapp/js/service/bbsReplyService.js
//服务层
app.service('bbsReplyService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../bbsReply/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../bbsReply/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../bbsReply/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity,topicUuid){
		return  $http.post('../bbsReply/add.do?topicUuid='+topicUuid,entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../bbsReply/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../bbsReply/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../bbsReply/search.do?page='+page+"&rows="+rows, searchEntity);
	}
	//查找主题帖及回复
	this.findTopAndReplys=function(uuid){
		return $http.get('../bbsReply/findTopAndReplys.do?uuid='+uuid);
	}   
});
